src/lib/EventListener/RequestListener.php

```diff
@@ -16,7 +15,12 @@
 
 class RequestListener implements EventSubscriberInterface
 {
-    use ContainerAwareTrait;
+    private $groupsBySiteAccess;
```

Comment:
As far as I understand this is an array, please enclose type hinting information.